extensions/strategies/trendline/strategy.js

```diff
@@ -3,46 +3,66 @@ var trend = require('trend')
+
     getOptions: function () {
-      this.option('period', 'period length, same as --periodLength', String, '1s')
-      this.option('periodLength', 'period length, same as --period', String, '1s')
+      this.option('period', 'period length', String, '30s')
+      this.option('periodLength', 'period length', String, '30s')
       this.option('lastpoints', "Number of trades for short trend average", Number, 100)
       this.option('avgpoints', "Number of trades for long trend average", Number, 1000)
       this.option('lastpoints2', "Number of trades for short trend average", Number, 10)
       this.option('avgpoints2', "Number of trades for long trend average", Number, 100)
-      this.option('min_periods', "Basically avgpoints + a BUNCH of more preroll periods for anything less than 5s period", Number, 5000)
-      this.option('max_sell_loss_pct', "Max Sell loss Pct", Number, 0)
-      this.option('markup_pct', "Default Strategy Markup - Hard In The Paint Mode", Number, 0.01)
-
+      this.option('min_periods', "Basically avgpoints + a BUNCH of more preroll periods for anything less than 5s period", Number, 15000)
+      this.option('markup_sell_pct', "test", Number, 0)
+      this.option('markdown_buy_pct', "test", Number, 0)
     },
+
     calculate: function (s) {
       get('lib.ema')(s, 'trendline', s.options.trendline)
       var tl1 = []
-      if (s.lookback[(s.options.min_periods)]) {
-        for (let i = 0; i < (s.options.min_periods); i++) { tl1.push(s.lookback[i].close) }
+      var tls = []
+      var tll = []
+      if (s.lookback[s.options.avgpoints + 2000]) {
+        for (let i = 0; i < s.options.avgpoints + 1000; i++) { tl1.push(s.lookback[i].close) }
+        for (let i = 0; i < s.options.lastpoints; i++) { tls.push(s.lookback[i].close) }
+        for (let i = 0; i < s.options.avgpoints; i++) { tll.push(s.lookback[i].close) }
+
           var chart = tl1
+
           growth = trend(chart, {
               lastPoints: s.options.lastpoints,
               avgPoints: s.options.avgpoints,
-              avgMinimum: 10,
+              avgMinimum: 0,
               reversed: true
-         }),
+          }),
           growth2 = trend(chart, {
               lastPoints: s.options.lastpoints2,
               avgPoints: s.options.avgpoints2,
-              avgMinimum: 10,
+              avgMinimum: 0,
               reversed: true
-         }),
-         s.stats = growth
-         s.growth = growth > 1
-         s.stats2 = growth2
-         s.growth2 = growth2 > 1
+          }),
+
+          s.stats = growth
+          s.growth = growth > 1
+          s.stats2 = growth2
+          s.growth2 = growth2 > 1
+          s.stdevs = stats.stdev(tls)
+          s.stdevl = stats.stdev(tll)
+          s.means = math.mean(tls)
+          s.meanl = math.mean(tll)
+          s.pcts = s.stdevs / s.means
+          s.pctl = s.stdevl / s.meanl
+          s.options.markup_sell_pct = math.mean(s.pcts, s.pctl) * 100
```

Comment:
This basically means the sell and buy price are set to a value that the trade never gets executed.  This also overwrites any defaults set to markup_sell pct give at command line.